custom_components/plugwise/__init__.py

```diff
@@ -88,7 +93,7 @@ async def update_listener(
     await hass.config_entries.async_reload(entry.entry_id)
 
 async def async_unload_entry(hass: HomeAssistant, entry: PlugwiseConfigEntry) -> bool:
-    """Unload a config entry."""
+    """Unload the Plugwise components."""
```

Comment:
Also here: `"""Unload the Plugwise Integration.`? Or `""" Unload the Plugwise platforms.`? Or `"""Unload the Plugwise config entry.`? The last two seem to make the most sense, to me :)

Reply:
Upload plugwise (also followed Core on this one)

Reply:
`Unload the Plugwise config entry`? Fits nicely with the `async_setup_entry()` docstring.